lib/ts/ink_res_init.cc

```diff
@@ -503,7 +503,20 @@ ink_res_init(ink_res_state statp,         ///< State object to update.
           hints.ai_family = PF_UNSPEC;
           hints.ai_socktype = SOCK_DGRAM; /*dummy*/
           hints.ai_flags = AI_NUMERICHOST;
-          sprintf(sbuf, "%d", NAMESERVER_PORT);
+
```

Comment:
I started looking around-- and it seems like that would work-- but I can't quite get it to work. `ats_ip_parse` gives me back a string, where I really want a sockaddr-- but when i use `ats_ip_pton` it doesn't seem to work for me-- either way this is how it was done, so IMO shouldn't be blocking-- although it would be good to clean that up.